modules/os2forms_nemid/src/Element/MitidChildrenSelectAjaxBehaviour.php

```diff
@@ -59,7 +59,7 @@ public static function mitidChildrenSelectAjax(array &$form, FormStateInterface
 
           switch ($element['#type']) {
             case 'os2forms_mitid_child_name':
-              $element['#value'] = !$cprLookupResult->isNameAddressProtected() ? $cprLookupResult->getName() : 'Navne- og adressebeskyttet';
+              $element['#value'] = !$cprLookupResult->isNameAddressProtected() ? $cprLookupResult->getName() : t('Name and address protected');
```

Comment:
You should use `StringTranslationTrait` and `$this->t` (rather than `t`) if possible.

Reply:
@rimi-itk 
generally agree, but I could not change it in the specified file - it's used as static function, therefore using $this is impossible.
Changed it in other places/files where possible.

please check again

Reply:
Makes sense, @stanbellcom. One final comment: https://github.com/OS2Forms/os2forms/pull/111#discussion_r1673816493